packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-sdk.ts

```diff
@@ -137,7 +137,10 @@ export interface ButtonsOptions {
     style?: PaypalButtonStyleOptions;
     fundingSource?: string;
     createOrder?(): Promise<string | void>; // TODO: this method should return only Promise<void>
-    onApprove?(data: ApproveDataOptions): void;
+    onApprove?(data: ApproveCallbackPayload, actions?: ApproveCallbackActions): void;
```

Comment:
Do we need `ApproveDataOptions` interface? (you updated `onApprove` data prop interface from `ApproveDataOptions` to `ApproveCallbackPayload`)

Reply:
Ping